Server/server/models/Player.ts

```diff
@@ -0,0 +1,36 @@
+import WebSocket from 'ws';
+
+export default class Player {
+    private blaster: WebSocket;
+    private vest: WebSocket;
+    private team: string;
+
```

Comment:
Add id as a property. 'B01' and 'S01' should correspond to player 1. Make id a number. Should also add a property to track whether player is alive or not (for deciding if messages from blaster/vest is valid or not.